workspace.go

```diff
@@ -1436,7 +1481,7 @@ func (o WorkspaceCreateOptions) valid() error {
 	if o.AgentPoolID == nil && (o.ExecutionMode != nil && *o.ExecutionMode == "agent") {
 		return ErrRequiredAgentPoolID
 	}
-	if o.TriggerPrefixes != nil && len(o.TriggerPrefixes) > 0 &&
+	if len(o.TriggerPrefixes) > 0 &&
```

Comment:
This change had me do a double take.  I didn't realize nil slices allowed calls to `len`!
You can't do that with other nils. TIL.